bucket/zoom.json

```diff
@@ -9,21 +9,20 @@
     "architecture": {
         "64bit": {
             "url": "https://cdn.zoom.us/prod/5.12.2.9281/x64/ZoomInstallerFull.exe#/dl.7z",
-            "hash": "f7cba2e790a7d41658ea0f430ac775f08d9872add59168fd72f12262bc9f9786"
+            "hash": "f7cba2e790a7d41658ea0f430ac775f08d9872add59168fd72f12262bc9f9786",
+            "pre_install": "Remove-Item \"$dir\\Installer.exe\"; Expand-7zipArchive \"$dir\\ZoomFull_Sip.cab\" -Removal"
```

Comment:
```suggestion
            "pre_install": "Expand-7zipArchive \"$dir\\ZoomFull_Sip.cab\" -Removal"
```

Reply:
Why should I **not** remove the installer.exe file?